File: API/Lieu.php
<?php

class Lieu implements JsonSerializable {
    private $id;
    private $cooX;
    private $cooY;
    private $descLieu;
    private $nom;

    function __construct( $_id, $_cooX,$_cooY,$_descLieu,$_nom)
    {
        $this->id = $_id;
        $this->cooX = $_cooX;
        $this->cooY = $_cooY;
        $this->descLieu = $_descLieu;
        $this->nom = $_nom;
    }

    function getId(){
        return $this->_id;
    }

    function getCooX(){
        return $this->_cooX;
    }

    function getCooY(){
        return $this->_cooY;
    }

    function getdesclieu(){
        return $this->_descLieu;
    }

    function getnom(){
        return $this->_nom;
    }

    public function jsonSerialize (){
        return [
            "id" => $this->id,
            "cooX" => $this->cooX,
            "cooY" => $this->cooY,
            "descLieu" => $this->descLieu,
            "nom" => $this->nom,
        ];
    }
}

File: Desktop/copiematete/matete-tommy-Matetest/Matetest/app/src/main/java/com/example/matetest/Lieu.java
package com.example.matetest;

public class Lieu {

    private int id;
    private double cooX;
    private double cooY;
    private String descLieu;
    private String nom;

    public Lieu(int id, double cooX, double cooY,String descLieu, String nom){
        this.id = id;
        this.cooX = cooX;
        this.cooY = cooY;
        this.descLieu = descLieu;
        this.nom = nom;
    }

    public int getId(){
        return id;
    }
    public double getCooX(){
        return cooX;
    }
    public double getCooY(){
        return cooY;
    }
    public String getDescLieu(){
        return descLieu;
    }
    public String getNom(){
        return nom;
    }

}
